packages/stark-core/src/modules/http/builder/http-abstract-base-request-builder.ts

```diff
@@ -12,8 +12,9 @@ export abstract class StarkAbstractHttpBaseRequestBuilder<T extends StarkResourc
 		this.request = request;
 	}
 
-	public setHeader(name: string, value: string): this {
-		if (typeof value !== "undefined") {
+	public setHeader(name: string, value: string | string[]): this {
+		// in Angular, a header value can only be string or string[], not null/undefined (https://github.com/angular/angular/issues/18743)
+		if (typeof value !== "undefined" && value !== null) {
```

Comment:
I understand that we cannot set undefined or null as header value but we don't check the name itself.
What would happen if the name was undefined or null ?

Reply:
True, I'll fix this